### PR TITLE
Fixed IncludeInPage with resource binding

### PR DESCRIPTION
**Pull request by adamjez**
Resource binding inside `IncludeInPage` throws `NullReferenceException` in `Render` method.

When the binding is evaluated to false we shouldn't render the control at all.

### CODE REVIEW

**Comment by exyi** [comment]
Hmm, the hardcoded values (and resource binding) are still disabled in `IncludeInPage` property https://github.com/riganti/dotvvm/blob/75ef61cd1be816b00fc3bc22e2177211a12a8f84/src/DotVVM.Framework/Controls/DotvvmControl.cs#L107 Is the check broken?

File: src/DotVVM.Framework/Controls/DotvvmControl.cs [hunk 1]
```diff
@@ -213,6 +213,11 @@ protected void AddDotvvmUniqueIdAttribute()
         /// </summary>
         protected virtual void RenderControl(IHtmlWriter writer, IDotvvmRequestContext context)
         {
+            if (HasBinding<ResourceBindingExpression>(IncludeInPageProperty) && !IncludeInPage)
```
[the remark this answers is not in the document]
Comment (exyi) [reply src/DotVVM.Framework/Controls/DotvvmControl.cs]: You would probably want to do the same for hardcoded values. But yes, this is probably the way it should behave.

**Comment by adamjez** [comment]
> Is the check broken?

Probably yes, resource binding is not considered as **HardCoded** value.

**Comment by exyi** [comment]
> > Is the check broken?
> 
> Probably yes, resource binding is not considered as **HardCoded** value.

Hmm, it not even considered as a binding, so it get's through any check...
I think we want to consider it as a hard-coded value, since it's basically
a "dynamic" value that should behave as normal assignment.

File: src/DotVVM.Framework/Controls/DotvvmControl.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (adamjez) [reply src/DotVVM.Framework/Controls/DotvvmControl.cs]: so should I enable `IncludeInPage` for **hardcoded** values?

**Pull request merged**